AllReadyApp/Web-App/AllReady.UnitTest/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarmShould.cs

```diff
@@ -26,25 +27,36 @@ public SendRequestStatusToGetASmokeAlarmShould()
         }
 
         [Fact]
-        public void SendJsonService()
+        public void SendsValidHttpRequestMessage()
         {
-            HttpRequestMessage requestMsg = null;
             var mockedHttpClient = new Mock<IHttpClient>();
-            mockedHttpClient.Setup(x => x.SendAsync(It.IsAny<HttpRequestMessage>()))
-                .ReturnsAsync(new HttpResponseMessage())
-                .Callback<HttpRequestMessage>(request => requestMsg = request);
             var sendRequestStatusToGetASmokeAlarm = new SendRequestStatusToGetASmokeAlarm(_getASmokeAlarmApiSettings, mockedHttpClient.Object);
             try
             {
                 sendRequestStatusToGetASmokeAlarm.Send("request1", "new", false);
             }
             catch { }
 
-            //mockedHttpClient.Verify(x => x.SendAsync(
-            //    It.Is<HttpRequestMessage>(request =>
-            //        request.RequestUri == new Uri("https://demo.getasmokealarm.org/admin/requests/status/request1")
-            //)));
+            mockedHttpClient.Verify(x => x.SendAsync(
+                It.Is<HttpRequestMessage>(request =>
+                    request.RequestUri == new Uri("https://demo.getasmokealarm.org/admin/requests/status/request1")
+                    && request.Headers.Authorization.Scheme == "someToken"
+                    && request.Content.ReadAsStringAsync().Result == "{\"acceptance\":false,\"status\":\"new\"}"
+                    && request.Content.Headers.ContentType.ToString() == "application/json; charset=utf-8"
+            )));
+        }
 
+        [Fact]
+        public void ThrowsException_WhenOnNonSuccessHttpStatus()
+        {
+            var mockedHttpClient = new Mock<IHttpClient>();
+            mockedHttpClient.Setup(x => x.SendAsync(It.IsAny<HttpRequestMessage>()))
+                .ReturnsAsync(new HttpResponseMessage(HttpStatusCode.BadRequest));
+            var sendRequestStatusToGetASmokeAlarm = new SendRequestStatusToGetASmokeAlarm(_getASmokeAlarmApiSettings, mockedHttpClient.Object);
+            
+            Assert.Throws<HttpRequestException>(
+                () => sendRequestStatusToGetASmokeAlarm.Send("request1", "new", false)
```

Comment:
please replace all the args to `.Send` with `It.IsAny<T>`. We don't care what the values are, the values being something explicit makes the unit test a little unclear as to whether we're relying on those values being passed in to be part of the condition that throws the exception.